src/components/OutputContainer/components/TransactionOutput.tsx

```diff
@@ -1,19 +1,18 @@
-import {
-  TRANSACTIONS_ENDPOINT,
-  ACCOUNTS_ENDPOINT
-} from '@multiversx/sdk-dapp/apiCalls/endpoints';
-import { useGetNetworkConfig } from '@multiversx/sdk-dapp/hooks/useGetNetworkConfig';
-import { ExplorerLink } from '@multiversx/sdk-dapp/UI/ExplorerLink';
 import { Label } from 'components/Label';
-import { FormatAmount } from 'components/sdkDappComponents';
 import { SignedTransactionType } from 'types';
+import { ExplorerLink } from '../../ExplorerLink';
+import {
+  ACCOUNTS_ENDPOINT,
```

Comment:
endpoins should be in sdk-dapp-core or utils

Reply:
They are imported from sdk-dapp-core